orb/commands/e2e_start_kind_cluster.yml

```diff
@@ -14,7 +14,8 @@ steps:
       command: |
         curl -LO https://github.com/kubernetes-sigs/kind/releases/download/v<< parameters.kind_version >>/kind-linux-amd64
         chmod +x kind-linux-amd64
-        mv kind-linux-amd64 /usr/local/bin/kind
+        mv kind-linux-amd64 kind
+        export PATH=$(pwd):$PATH
```

Comment:
What is the value of `$(pwd)` at this point? Is it just the home dir of the user running the container?

Reply:
I believe it is, yes. I've updated it to create a new `bin` dir, but now I'm wondering if i should make it even more unique. Some containers may have a `bin` dir already.